test/sagemaker_tests/autogluon/training/conftest.py

```diff
@@ -282,6 +286,21 @@ def skip_gpu_instance_restricted_regions(region, instance_type):
         pytest.skip("Skipping GPU test in region {}".format(region))
 
 
+@pytest.fixture(autouse=True)
+def skip_p5_tests(request, ecr_image, instance_type):
+    if "p5." in instance_type:
+        framework, image_framework_version = get_framework_and_version_from_tag(ecr_image)
+
+        image_processor = get_processor_from_image_uri(img_uri)
+        image_cuda_version = get_cuda_version_from_tag(ecr_image)
+        if (
+            image_processor != "gpu"
+            or Version(image_framework_version) in SpecifierSet("<2.0.1")
```

Comment:
These are AutoGluon DLCs, and have a different versioning structure.

I would recommend only checking cuda version for these, instead.